### DISCUSSION TITLE
POST method doesn’t see token

### DISCUSSION THREAD

**Discussion by malogajski** [dedoc/scramble#271] [Q&A]
I have a issue when using the POST method. It looks to me like the api token is not being passed. However if I change to GET everything works as expected.
I tested with Postman and everything works in every variant. I also tried to remove the api token from postman and then I got the same result as when I test through the documentation with the POST method, which is that it redirects me to the login frontend page.

I'm not sure if I overlooked something or if this is a bug?! Please check example:
API routes:
```
Route::group([
    'prefix' => 'v1',
    'as' => 'api.',
    'middleware' => ['auth.access', 'auth:api'],
], function () {
    // Post
    Route::get('post/create', [PostApiControllerController::class, 'store']);
    Route::get('post', [PostApiControllerController::class, 'index']);
});
```

Store function from controller:
```
/**
* Create
*
* Create new post
*
* @authetnicated
*
*/
public function store(Request $request)
{
    return response()->json(['test']);
}
```
Solution above works, but if I change "create" endpoint to POST api token not available when checking via documentation. Of course it works with Postman, even if route endpoint for both method are same but with different methods:
```
Route::get('post', [PostApiControllerController::class, 'index']); 
Route::post('post', [PostApiControllerController::class, 'store']); 
```

Edit:
I see the point. With GET, parameters are "query" type, and with POST they are "body". So what I need here is an annotation for the parameters in order to set them as a query since api token is in both cases as "query" parameter.

**Comment by romalytvynenko** [comment]
@malogajski yup, there is support for `query` "placement" parameter in future versions.
